src/main/java/mdlaf/components/titlepane/MaterialTitlePaneUI.java

```diff
@@ -514,14 +514,14 @@ protected void paintComponent(Graphics g) {
                     rect.x = window.getWidth() - window.getInsets().right - 2;
                 }
                 titleW = rect.x - xOffset - 4;
-                theTitle = SwingUtilities2.clipStringIfNecessary(rootPane, fm, theTitle, titleW);
+                theTitle = BasicGraphicsUtils.getClippedString(rootPane,fm, theTitle, titleW);
             } else {
                 titleW = xOffset - rect.x - rect.width - 4;
-                theTitle = SwingUtilities2.clipStringIfNecessary(rootPane, fm, theTitle, titleW);
-                xOffset -= SwingUtilities2.stringWidth(rootPane, fm, theTitle);
+                theTitle = BasicGraphicsUtils.getClippedString(rootPane,fm, theTitle, titleW);
```

Comment:
I don't understand because you have used `BasicGraphicsUtils.getClippedString ` this functions isn't included inside the JDK 8.
Also f my preventive quarantine chamber doesn’t make me cloudy this change permitted the execution of the material library with JDK >= 9, right? if yes, I think this change isn't permitted inside the library because a lot of projects is using still the JKD7 and JDK8.

Ex: JMars(software used to NASA) used JDK7 or JDK8 I don't remember the correct version but is under JDK9 or the library is presented at an OOP course at the University of Basilicata (Italian University) and it uses JDK8.

Please fix me if I'm loading somethings.

Reply:
This may be related to:
https://github.com/bulenkov/Darcula/issues/23

I think you can't use SwingUtilities2 in never JVM versions, so we will need to update at some point?

Reply:
I think yes, but then my reflection now is "material-ui-swing is an l&f to increase the style of swing, and the swing is an old framework for Java GUI, I think if now we update the library to java >= 9 we can lose a lot of users that use the library inside the project."

I don't want to leave users without a library with the material style, the effort that I am making in these years of development on this library and support all versions of java even with small warnings.

Reply:
OK, I think it is possible to maintain support for Java 8 while also making it compatible with Java 9+. We just need to find alternatives for the internal sun APIs (possibly we can simply duplicate code from OpenJDK8?)

Reply:
We can copy all code inside the SwingUtils2 because if I remember well, the code method `clipStringIfNecessary` called other methods inside the class.

But I think if you restore the change, so you pass to BasicGraphicsUtils at SwingUtilities2, you are able to compile the code of the library.
You should receive only a warning error.

P.S: The retro compatibly of the library is very important for me and for the library.

Reply:
OK, will restore this and update the PR.